static/app/utils/replays/hooks/useReplayData.tsx

```diff
@@ -140,6 +141,34 @@ function useReplayData({
         `/organizations/${orgSlug}/replays-events-meta/`,
         {
           query: {
+            dataset: DiscoverDatasets.DISCOVER,
+            start: replayRecord?.started_at.toISOString(),
+            end: finishedAtClone.toISOString(),
+            project: ALL_ACCESS_PROJECTS,
+            query: `replayId:[${replayRecord?.id}]`,
+            per_page,
+            cursor,
+          },
+        },
+      ];
+    },
+    [orgSlug, replayRecord]
+  );
+
+  const getPlatformErrorsQueryKey = useCallback(
+    ({cursor, per_page}): ApiQueryKey => {
+      // Clone the `finished_at` time and bump it up one second because finishedAt
+      // has the `ms` portion truncated, while replays-events-meta operates on
+      // timestamps with `ms` attached. So finishedAt could be at time `12:00:00.000Z`
+      // while the event is saved with `12:00:00.450Z`.
+      const finishedAtClone = new Date(replayRecord?.finished_at ?? '');
+      finishedAtClone.setSeconds(finishedAtClone.getSeconds() + 1);
+
+      return [
+        `/organizations/${orgSlug}/replays-events-meta/`,
+        {
+          query: {
+            dataset: DiscoverDatasets.ISSUE_PLATFORM,
             start: replayRecord?.started_at.toISOString(),
             end: finishedAtClone.toISOString(),
             project: ALL_ACCESS_PROJECTS,
```

Comment:
noticed that the other `QueryKey` has these extra params as well -- do we need them here too?

```
query: `replayId:[${replayRecord?.id}]`,
            per_page,
            cursor,`
```

Reply:
yeah, should be.. hrm

Reply:
it's all good, those props do exist in both the places where we call `useFetchSequentialPages`: inside `getErrorsQueryKey` and inside `getPlatformErrorsQueryKey`.

I think the diff algorithm is picking hard to read chunks, but expanding all the lines in the file, and also using Unified diff mode makes it better. 
![SCR-20240312-mfyn](https://github.com/getsentry/sentry/assets/187460/b8387e4a-4e08-4896-9612-62c9bc707a36)

Can also look at the file on the branch directly: 
![SCR-20240312-nncq](https://github.com/getsentry/sentry/assets/187460/363db647-acc5-4ab4-9eb7-982aaf01c6b3)

https://github.com/getsentry/sentry/blob/8af2f5ce1a742854ed0530776f9e3753a55da23c/static/app/utils/replays/hooks/useReplayData.tsx

Reply:
oh nice! didn't realize, that's my bad

Reply:
nbd! i had to look twice too